app/views/users/notes/_search.html.slim

```diff
@@ -0,0 +1,9 @@
+.top-bar
+  = simple_form_for :search_form,
+                    method: :get, url: action,
+                    html: { class: "menu" } do |f|
+    li
+      = f.input :query, label: false, placeholder: "Search notes",
+                input_html: { value: params.fetch(:search_form, {})[:query] }
```

Comment:
```suggestion
                input_html: { value: filter_params[:query] }
```

You can use helper method in the controller. In this case, you don't have to read parameters
Notes controller:
```ruby
class NotesController < ApplicationController
  expose_decorated :notes, :fetch_notes
  
  helper_method :filter_params
  ...
  def filter_params
    params.require(:search_form).permit(:query).to_h
  end
end